kube-client/src/config/file_loader.rs

```diff
@@ -83,12 +83,21 @@ impl ConfigLoader {
             .ok_or_else(|| KubeconfigError::LoadClusterOfContext(cluster_name.clone()))?;
 
         let user_name = user.unwrap_or(&current_context.user);
+
+        // client-go doesn't fail on empty/missing user, so we don't either
+        // see https://github.com/kube-rs/kube/issues/1594
         let mut user = config
             .auth_infos
             .iter()
             .find(|named_user| &named_user.name == user_name)
             .and_then(|named_user| named_user.auth_info.clone())
-            .ok_or_else(|| KubeconfigError::FindUser(user_name.clone()))?;
```

Comment:
We should remove this error variant as well in config/mod.

Reply:
Good catch